Shared/DataTransferObjects/AttendanceDto.cs

```diff
@@ -0,0 +1,9 @@
+namespace Shared.DataTransferObjects;
+
+public record AttendanceDto(
+    Guid Id, 
+    Guid EmployeeId, 
+    DateTime CheckInTime, 
+    DateTime? CheckOutTime, 
+    string Status, 
```

Comment:
Consider making the `Status` property nullable or defining a constrained set of possible string values (e.g., using an enum) if applicable. This would improve clarity and enforce stricter typing.

**Suggested change**
```suggestion
    string? Status,
```